app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name = command.split()
+    if (name[0] == "cp"
+            and name[1] != name[2]):
```

Comment:
no need to take values by index, u can use split method to unpack splitted string to variables
`variable1, variable2, variable3= ur_string.split()`